### DISCUSSION TITLE
multisite mounting

### DISCUSSION THREAD

**Discussion by paulhaem** [statamic/cms#4848] [Q&A]
Hello,
how does mounting of collections work in a multisite scenario. Currently I am only able to choose a single site (e.g. "English"). The [docs ](https://statamic.dev/collections#mounting) are not going into details regarding this scenario. The selection of the site is quite vague due to the missing `Site` or `Locale` info in the list (already discussed in #4092).

**Comment by edalzell** [comment]
What are you trying to do?

**Reply by paulhaem** [reply]
I have a multisite with a site for each locale. Now I would like to mount the `Blog` collection on to the blog page to make the URL dependant of the page and give the users the possibility to add entries via the pages collection. Latter is only available on the specific page I mounted the collection onto, the origin page entry `Blog`. When switching to other sites / locales I am not able to see the "Add / Edit" buttons on the site specific `Blog` page entry.

**Comment by mzdr** [comment]
Is there any progress on this one? I am trying to migrate from v2 to v3 and I am having a hard time mounting a collection to a page in a multisite setup as well. ~For me it's not even working when I try to set a mount on the default locale page that I want it mounted to. It just doesn't show Add/Edit even though its mounted.~ (Had List view instead of Tree)

#6906 seems to be not progressing as well. Any news on this topic?

Edit: I think I managed it via Navigations. I created one myself and manually added the items. It's quite a manual process, but solves my issue for now.
